src/assets/SASSAsset.js

```diff
@@ -43,6 +43,11 @@ class SASSAsset extends Asset {
     opts.importer = (url, prev, done) => {
       let resolved;
       try {
+        if (!url.match(/^(~|\.\/|\/)/)) {
+          url = './' + url;
+        } else if (!url.match(/^(~\/|\.\/|\/)/)) {
+          url = url.substring(1);
+        }
```

Comment:
[`.test` is more efficient](https://jsperf.com/test-vs-match-regex) and more suited

Reply:
Awesome! Thanks for testing that out